modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entry/image_selector.jspf

```diff
@@ -57,74 +57,94 @@ long fileEntryId = BeanParamUtil.getLong(cpAttachmentFileEntry, request, "fileEn
 	</c:when>
 </c:choose>
 
-<aui:script use="liferay-item-selector-dialog">
-	var selectFileButton = window.document.querySelector(
-		'#<portlet:namespace />selectFile'
+<aui:script sandbox="<%= true %>">
+	const selectFileButton = document.getElementById(
+		'<portlet:namespace />selectFile'
 	);
 
 	if (selectFileButton) {
```

Comment:
Wouldn't we prefer to return early rather than wrap the whole eventHandler block?